locast2dvr/dvr.py

```diff
@@ -203,3 +207,29 @@ def _remap(station: dict, i: int):
         new_channel = str(float(station['channel']) + 100 * i)
 
     return (new_channel, station['callSign'].replace(station['channel'], new_channel))
+
+def _remap_by_name(stations):
```

Comment:
I've been thinking about the approach here a bit. Not that it matters that much, but if we would like to remap in a more generic way, we could split this algorithm into two parts: one build a map of how you want things to be remapped and then do the actual remapping. 

You could think of something of first creating a map that looks somewhat like this:
```
callsign_to_channels = {
  "CBS": [station0, station4, station10],
  "PBS": [station2, station5, station15],
  ...
}
```

Sort this map and then feed that to a function that does the actual remapping. This could even be a generalized function that works for multiple scenarios.

For the current remap, we'd construct a map that looks like:
```
dvr_to_channels = {
  0: [station0, station4, station10],
  1: [station2, station5, station15],
  ...
}
```

Reply:
For "station#", would that match up with the "id" value from the dictionaries in the stations list?